egs/libri_2_mix/ConvTasNet/local/conf.yml

```diff
@@ -0,0 +1,38 @@
+encoder:
+  enc_name : free
+decoder:
+  dec_name : free
+
+# filterbank config
+filterbank:
+  n_filters: 512
+  kernel_size: 32
+  stride: 16
+# Network config
+masknet:
+  n_blocks: 6
+  n_repeats: 2
+  mask_act: relu
+  bn_chan: 128
+  skip_chan: 128
+  hid_chan: 512
+# Training config
+training:
+  epochs: 200
+  batch_size: 8
+  num_workers: 2
+  half_lr: yes
+  early_stop: yes
+# Optim config
+optim:
+  optimizer: adam
+  lr: 0.001
+  weight_decay: 0.
+# Data config
+data:
+  metadata_train_path : "/home/jcosentino/libri/libri2mix/8K/min/metadata/mixture_train-clean-100.csv"
+  metadata_val_path : "/home/jcosentino/libri/libri2mix/8K/min/metadata/mixture_dev-clean.csv"
```

Comment:
We're trying that only `run.sh` needs to be modified when running the recipe. We could create a local `./data` folder and copy things there? 
Let's try to avoid needing modifs in the conf file